.circleci/config.yml

```diff
@@ -843,7 +843,7 @@ jobs:
           command: |
             curl -L https://foundry.paradigm.xyz | bash
             source $HOME/.bashrc
-            foundryup
```

Comment:
This is going to make it very slow because it will build forge, we should not do this

Reply:
We probably should be using the "correct" commit of foundry though no?

Reply:
I reverted this change as it seems to be working now.
If/when it breaks for being the wrong version we can handle that separately.